server/data/breachNoticeApiClient.ts

```diff
@@ -104,34 +112,25 @@ export interface SelectItem {
   selected: boolean
 }
 
-export interface RadioButton {
-  value: string
-  text: string
-  checked: boolean
+// Reference Data
+export interface ReferenceData {
+  code: string
+  description: string
 }
 
-export interface EnforceableContactRadioButton {
-  datetime: string
-  type: ReferenceData
-  outcome: ReferenceData
-  notes: string
-  requirement: Requirement
-  checked: string
+export interface RadioButton {
```

Comment:
Minor thing, feel free to ignore - but would it help organise things if these Radio/SelectItem/ErrorMessage types were moved into their own `uiModels.ts` file, given they don't directly relate to the APIs?